src/NumberInput/NumberInput.tsx

```diff
@@ -57,6 +57,9 @@ type Props = {
   step: number;
```

Comment:
@txwkx `step` isn't optional in the type declaration, but if I omit it from the component then nothing happens...no warnings, or anything. Any idea why?